src/gobstuf/stuf/brp/request/ingeschrevenpersonen.py

```diff
@@ -6,6 +6,35 @@ class IngeschrevenpersonenStufRequest(StufRequest):
     content_root_elm = 'soapenv:Body BG:npsLv01'
     soap_action = 'http://www.egem.nl/StUF/sector/bg/0310/npsLv01Integraal'
 
+    BSN_LENGTH = 9
```

Comment:
Waarschijnlijk ten overvloede, en waarschijnlijk al een ei over gelegd, maar we accepteren geen 8-cijferige BSN's en verwachten dat de gebruiker van de API de 0 toevoegt? Hoe gaat MKS hiermee om?